src/common/models/Projection.ts

```diff
@@ -435,7 +432,9 @@ export class Projection {
     // TODO(michael): I don't like hardcoding "County" into the name. We should
     // probably delete this code and get the location name from somewhere other
     // than the API (or improve the API value).
-    return s.countyName ? `${s.countyName}, ${s.stateName}` : s.stateName;
+    // TODO(chris): Fix state
+    // @ts-ignore
+    return s.county ? `${s.county}, ${STATES[s.state]}]` : STATES[s.state];
```

Comment:
Perhaps add a getStateName() helper to locations.ts that either does the ts-ignore or `(STATES as any)[stateCode]`.

Alternatively, we have [getLocationNameForFips()](https://github.com/covid-projections/covid-projections/blob/062a2ac10fae43292e5c42eb34f48fbc69ab9f21/src/common/locations.ts#L119) we could use, but it only includes the state abbreviation.  We could change it to accept options or something like `getLocationNameForFips(s.fips, { useStateName: true })`